rules/explicit-length-check.js

```diff
@@ -33,5 +125,7 @@ const create = context => {
 
 module.exports = {
 	create,
-	meta: {}
```

Comment:
The option should be defined using a `schema` so it can be validated by ESLint

See: http://eslint.org/docs/developer-guide/working-with-rules#options-schemas